test/specs/basicAuth.spec.js

```diff
@@ -1,3 +1,46 @@
+import axios from '../../lib/axios'
+
+// Validate an invalid character error
+function validateInvalidCharacterError (error) {
+  expect(/character/i.test(error.message)).toEqual(true)
+}
+
 describe('basicAuth without btoa polyfill', function () {
-  setupBasicAuthTest()
+  beforeEach(function () {
+    jasmine.Ajax.install()
+  })
+
+  afterEach(function () {
+    jasmine.Ajax.uninstall()
+  })
+
+  it('should accept HTTP Basic auth with username/password', function (done) {
+    axios.get('/foo', {
+      auth: {
+        username: 'Aladdin',
```

Comment:
isn't `Alibaba` who said that 🙈

Reply:
Hahahaha, I just copy & 🍝 it, but yeah you're right 😂